helpers/inject.js

```diff
@@ -1,6 +1,37 @@
 'use strict';
 
 const factory = globals => {
+    function filterValues(value) {
+        let result = value;
+        try {
+            JSON.parse(value);
```

Comment:
Can we make sure we have test cases for all scenarios

Reply:
Building the test cases is a good idea, because while writing tests I found that I need to update my changes to account for Array specifically (we have some injectable values that are arrays nested in objects nested in arrays etc. like blogs).  😅

Reply:
I think it should be caught by the object check in the catch